geomesa-cassandra/geomesa-cassandra-tools/src/main/scala/org/locationtech/geomesa/tools/cassandra/commands/CommandWithCassandraDataStore.scala

```diff
@@ -0,0 +1,33 @@
+package org.locationtech.geomesa.tools.cassandra.commands
```

Comment:
two commands you might find useful:
`mvn license:check`
`mvn license:format`
be warned, the format will not always clean up the old header so double check it manually